Color Picker 구현

jasonkang14 opened the pull request:
1. Mid Socks 활용하여 ColorPicker구현
2. 명일 양말 실사 활용하여 4가지 타입 양말 Color Picker모두 구현 계획

Commit:
typo fixed

Commit:
color picker completed by using 2018 fashion colors to change background
color of socks based on a user's choice

typo fixed

background-image for a sock changed

image for a sock added

background image for a sock changed

wip

a separate js file created to import and export sock images

sock images added

minor changes to Button Component

color picker complete

Commit:
background-image for a sock changed

Commit:
image for a sock added

Commit:
background image for a sock changed

Yeri-Kim commented on src/Pages/Main/Custom/custom.scss:
```diff
@@ -6,9 +6,13 @@
+    width: 26.5em;
     height: 30em;
     border: 1px solid black;
+    background-image: url(../../../../src/assets/images/middleSocksClear.png);
```
절대경로 써주세요

jasonkang14 replied:
변경했습니다

Yeri-Kim commented on src/Pages/Main/Custom/custom.scss:
```diff
@@ -6,9 +6,13 @@
 
   .socksContainer {
-    width: 30em;
+    width: 26.5em;
```
em을 쓰신 이유는 무엇인가요?

jasonkang14 replied:
responsive할 때 수월하다고 해서 사용했습니다... 아닌가요??

Yeri-Kim replied:
responsive할 때 font-size를 화면 크기마다 조정하기 위해 em을 쓰는 경우는 많지만, 일반적인 요소의 가로세로를 지정할 때는 잘 쓰지 않습니다. 왜냐하면 em의 default 크기가 font-size에 기반한것이라서요.

jasonkang14 commented:
변경했습니다.

jasonkang14 commented:
다시 push 하겠습니다.

jasonkang14 commented:
수정사항 반영했습니다. (상대경로 -> 절대경로)

Commit:
conflicts resolved

Commit:
css fixed

Commit:
# 커밋 3개가 섞인 결과입니다.
# 1번째 커밋 메시지입니다:

color picker completed by using 2018 fashion colors to change background
color of socks based on a user's choice

typo fixed

background-image for a sock changed

image for a sock added

background image for a sock changed

wip

a separate js file created to import and export sock images

sock images added

minor changes to Button Component

color picker complete

# 커밋 메시지 #2번입니다:

background-image for a sock changed

# 커밋 메시지 #3번입니다:

image for a sock added

Commit:
# 커밋 4개가 섞인 결과입니다.tree 24bb2982c2402f186780a26bfa6e1f2e1587719a
parent bb7752e32fbe157e41c995ab8bfe7390cc71e47b
author Byeongjin Kang <byeongjinkang@Byeongjinui-MacBookPro.local> 1563417361 +0900
committer Byeongjin Kang <byeongjinkang@Byeongjinui-MacBookPro.local> 1563418292 +0900

# 커밋 3개가 섞인 결과입니다.
# 1번째 커밋 메시지입니다:

# 커밋 3개가 섞인 결과입니다.
# 1번째 커밋 메시지입니다:

color picker completed by using 2018 fashion colors to change background
color of socks based on a user's choice

typo fixed

background-image for a sock changed

image for a sock added

background image for a sock changed

wip

a separate js file created to import and export sock images

sock images added

minor changes to Button Component

color picker complete

# 커밋 메시지 #2번입니다:

background-image for a sock changed

# 커밋 메시지 #3번입니다:

image for a sock added

# 커밋 메시지 #2번입니다:

1. Front images for a sock filpped vertically
2. Units for CSS changed (em -> px)

# 커밋 메시지 #3번입니다:

typo fixed

# 커밋 메시지 #4번입니다:

image for a sock added

Commit:
1. Front images for a sock filpped vertically
2. Units for CSS changed (em -> px)

Commit:
conflicts resolved

Commit:
conflicts resolved after a rebase

Commit:
Merge branch 'feature/color_picker' of https://github.com/wecode-bootcamp-korea/we-socks-frontend into feature/color_picker

Merging the remote branch to the origin before making a pull request

Commit:
custom semi-complete
